### PR TITLE
perf: show auth in UI

### PR DESCRIPTION
**Pull request by yuqizhou77**

### CODE REVIEW

File: packages/fx-core/src/component/generator/copilotPlugin/helper.ts [hunk 1]
```diff
@@ -291,6 +291,13 @@ function sortOperations(operations: ListAPIInfo[]): ApiOperation[] {
       id: operation.api,
       label: operation.api,
       groupName: arr[0],
+      detail: !operation.auth
+        ? getLocalizedString("core.copilotPlugin.api.noAuth")
+        : Utils.isBearerTokenAuth(operation.auth.authScheme)
+        ? getLocalizedString("core.copilotPlugin.api.apiKeyAuth")
+        : Utils.isOAuthWithAuthCodeFlow(operation.auth.authScheme)
+        ? "OAuth"
```
Comment (SLdragon) [inline packages/fx-core/src/component/generator/copilotPlugin/helper.ts:299]: maybe "OAuth" should be also put into package.nls.json file?

Comment (yuqizhou77) [reply packages/fx-core/src/component/generator/copilotPlugin/helper.ts:299]: Does it need any localization?

Comment (yuqizhou77) [reply packages/fx-core/src/component/generator/copilotPlugin/helper.ts:299]: Looking at this https://learn.microsoft.com/zh-cn/entra/identity-platform/v2-oauth2-auth-code-flow OAuth is not localized.

**Review by jayzhang — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Comment by yuqizhou77** [comment]
@supkasar could you help review strings in this PR? Thanks

**Pull request merged**